docs/users-guide/saved-questions.md

```diff
@@ -0,0 +1,91 @@
+## Using a saved question as a sub-query
+
+With SQL databases, we can use an existing question as the basis for a new query, or as a common table expression (CTE). For example, let's say we have a lot of data spread across a number of tables, but our users are most interested in a subset of that data. We can perform a complicated query once to return those results, which people can refer to in their queries, just like they would refer to any other table.
+
+Here's how it works: create and save a question that returns the result set you'd like to make available for people to query. Using the sample dataset included with Metabase as an example, let's say we want to provide a result set (a "table") that only has orders from 2019, and only includes orders for products in the Gizmo category.
+
+We could create this question using the notebook editor, like so:
+
+![Example notebook](images/saved-questions/example-notebook.png).
+
+Alternatively, we could create that question using SQL:
+
+```
+SELECT *
+FROM   orders AS o
+       INNER JOIN products AS p
+               ON o.product_id = p.id
+WHERE  p.category = 'Gizmo'
+       AND o.created_at BETWEEN '2019-01-01' AND '2019-12-31'
+```
+
+We'll save that question as "Gizmo orders in 2019".
+
+Now let's refer to that "Gizmo orders in 2019" in a new query. To keep it simple, let's say we just want to count all of the Gizmo orders from 2019. We can use the `#` symbol to refer to a saved question in a query.
+
+If we type out:
+
+```
+SELECT count(*)
+FROM {% raw %}{{#{% endraw %}
+```
+
+Metabase will slide out a sidebar where we can select a question to reference. We'll search for our "Gizmo orders in 2019" question
+
+![Select a question from the variable sidebar](images/saved-questions/variable-sidebar.png)
+
+We'll select our "Gizmo orders in 2019" question, and Metabase will update our code with the question's ID, `5`:
+
+```
+SELECT count(*)
+FROM {% raw %}{{#5}}{% endraw %}
+```
+
+This query returns the number of records from our saved question, "Gizmo orders in 2019", which has an ID of `5`.
+
+## Saved question as a Common Table Expression (CTE)
+
+The same syntax can be used in [Common Table Expressions (CTEs)](https://en.wikipedia.org/wiki/Hierarchical_and_recursive_queries_in_SQL#Common_table_expression) (with SQL databases that support CTEs):
+
+```
+WITH 2019_gizmo_orders AS {% raw %}{{#5}}{% endraw %}
+SELECT count(*)
+FROM 2019_gizmo_orders
+```
+
+The `{% raw %}{{#5}}{% endraw %}` tag is substituted for the SQL query of the referenced question, surrounded by parentheses, equivalent to:
```

Comment:
I think this could be clarified:

```suggestion
When this query is run, the `{% raw %}{{#5}}{% endraw %}` tag will be substituted with the SQL query of the referenced question, surrounded by parentheses. So it'll look like this under the hood:
```